Recurly.Tests/Fixtures.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Recurly.Tests
+{
+    public class MyResource : Resource
+    {
+        [JsonProperty("my_string")]
+        public string MyString { get; set; }
+
+        [JsonProperty("my_float")]
+        public float? MyFloat { get; set; }
+
+        [JsonProperty("my_int")]
+        public int? MyInt { get; set; }
+
+        [JsonProperty("my_date_time")]
+        public DateTime? MyDateTime { get; set; }
+
+        [JsonProperty("my_sub_resource")]
+        public MySubResource MySubResource { get; set; }
+
+        [JsonProperty("my_array_string")]
+        public List<string> MyArrayString { get; set; }
```

Comment:
Does our API ever expose an array of anything besides strings and more resources? Like an array of date times or something crazy like that?

Reply:
not currently. Since this isn't really our code i feel like just testing a primitive is enough.